Selenium/CodingConventions.md

```diff
@@ -132,3 +169,32 @@ public class EventsArticleEditMode {
 
 }		
 ```
+
+## Other method rules
+
+### methods order
+
+Methods in file should have alphabeticall order. If You want to add next click method You should add it after clickSave method, not in the end of the file. When methods will be in alphabeticall order we can faster find them in big files.
```

Comment:
Is this kind of ordering necessary with modern IDEs?  I feel like its more important to have related methods near each other.  That way the reader can keep more relevant code on screen at once to understand what code is doing, rather than jumping back and forth through a file.

This section on ```Vertical Distance``` of Clean Code (Page 80, https://cleansourcecode.files.wordpress.com/2013/10/clean-code.pdf) sums it up nicely for variables and functions.